src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -0,0 +1,571 @@
+      </td>
+      <td>
+HTTP Range requests are not compatible
+      </td>
+    <tr>
+      <td>
+Compatible with MSE and segment files
+      </td>
+      <td>
+Should be used only for audio files or short video clips when fetching full
+resource
+      </td>
+    </tr>
+    <tr>
+      <td>
+<a href="#manual_buffering">Manual buffering</a>
+      </td>
+      <td>
+Full control
+      </td>
+      <td>
+Complex error handling is website responsibility
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Video preload attribute
+
+If the video source is an unique file hosted on a web server, you may want to
+use the video `preload` attribute to provide a hint to the browser as to [how
+much information or content to preload]. This means [Media Source Extensions
+(MSE)] is not compatible with `preload`.
+
+Resource fetching will start only when initial HTML document has been
+completely loaded and parsed (eg. `DOMContentLoaded` event) while the very
+different `load` event will be fired when resource has actually been fetched.
+
+<figure class="TODO">
+  <img src="/web/fundamentals/media/images/video-preload/video-preload.svg">
+</figure>
+
+Setting `preload` value to `metadata` indicates that the user is not expected
+to need the video, but that fetching its metadata (dimensions, track list,
+duration, and so on) is desirable.
+
+```
+<video id="video" preload="metadata" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('progress', function() {
+    if (video.buffered.length === 0) return;
+
+    var bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(bufferedSeconds + ' seconds of video are ready to play!');
+  });
+</script>
+```
+
+Setting `preload` value to `auto` indicates that the browser should cache
+enough such that the playback to end would be possible without requiring a stop
+for further buffering.
+
+```
+<video id="video" preload="auto" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('progress', function() {
+    if (video.buffered.length === 0) return;
+
+    var bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(bufferedSeconds + ' seconds of video are ready to play!');
+  });
+</script>
+```
+
+There are some caveats though. As this is just a hint, browser may completely
+ignore the `preload` attribute. At the time of writing, here are some rules
+applied in Chrome:
+
+- When [Data Saver] is enabled, Chrome forces `preload` value to `none`.
+- In Android 4.3, Chrome forces `preload` value to `none`.
+- On a cellular connection (2G, 3G, and 4G), Chrome forces `preload` value to
+  `metadata`.
+
+### Tips
+
+If your website contains plenty of video resources on the same domain, I would
+recommend you set the `preload` value to `metadata` or define the `poster`
+attribute and set `preload` value to `none`. That way, you would avoid hitting
+the maxmium number of HTTP connections per host (6 in Chrome at the time of
```

Comment:
I won't call Chrome specifically then. Thank you!

For info the HTTP 1.1 spec says:

```
   Clients that use persistent connections SHOULD limit the number of
   simultaneous connections that they maintain to a given server. A
   single-user client SHOULD NOT maintain more than 2 connections with
   any server or proxy. A proxy SHOULD use up to 2*N connections to
   another server or proxy, where N is the number of simultaneously
   active users. These guidelines are intended to improve HTTP response
   times and avoid congestion.
```